src/lib.rs

```diff
@@ -140,21 +139,94 @@ impl Wiki {
 
         Ok(())
     }
+    /// read all the files from current paths
```

Comment:
This description not really describes what this function actually does (and to be honest, i have no idea too)